ivre/tools/db2view.py

```diff
@@ -71,18 +71,43 @@ def my_parse_args():
             exit(-1)
     else:
         subparsers = parser.add_subparsers(dest='view_source',
-                                           help="Accepted values are "
-                                                "'nmap' and 'passive'")
+                                           help="Accepted values are 'nmap' "
+                                                "and 'passive'. None or 'all' "
+                                                "will do both")
 
         subparsers.add_parser('nmap', parents=[db.nmap.argparser])
         passparser = subparsers.add_parser('passive',
                                            parents=[db.passive.argparser])
         passparser.add_argument('ips', nargs='*')
+        subparsers.add_parser('all')
 
     args = parser.parse_args()
 
-    if args.category:
-        db.view.category = args.category
+    if args.view_category:
+        db.view.category = args.view_category
```

Comment:
Didn't we conclude that this could not be a `db.view` attribute (thread / fork unsafe + weird...)? Or do I forget something?

Reply:
This attribute is absolutely a concurrency issue. I'll get rid of this.

Reply:
So I think passing parameter is the best way here.